app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -28,6 +28,7 @@ module Api::V2::TaxonomiesController
       param :hostgroup_ids, Array, N_("Host group IDs"), :required => false
       param :environment_ids, Array, N_("Environment IDs"), :required => false
       param :subnet_ids, Array, N_("Subnet IDs"), :required => false
+      param :ancestry, String, N_("Parent ID"), :required => false
```

Comment:
(Also, ancestry is a kind of internal field, it stores the whole ancestry and not just the parent.)